packages/webcert/src/components/modal/CareProviders.tsx

```diff
@@ -0,0 +1,21 @@
+import { CareProvider } from '@frontend/common'
+import React from 'react'
+import { CareUnits } from './CareUnits'
+
+interface Props {
+  careProviders: CareProvider[]
+  chooseUnit: (event: React.MouseEvent) => void
+}
+
+export const CareProviders: React.FC<Props> = ({ chooseUnit, careProviders }) => {
+  return (
+    <>
+      {careProviders.map((careProvider) => (
+        <details className="ic-card ic-card--expandable ic-card--sm-unset-style ic-expandable" key={careProvider.id} open>
```

Comment:
Can the expandable text be extracted in a more generalized component similar to the modal?